stl/inc/xstring

```diff
@@ -1274,8 +1274,10 @@ public:
 
 #if _HAS_CXX23
     // clang-format off
-    template <_RANGES contiguous_range _Range>
-        requires (_RANGES sized_range<_Range>
+    template <class _Range>
+        requires (!same_as<remove_cvref_t<_Range>, basic_string_view>

```

Comment:
Looks good. Should we cite `// per LWG-NNNN` if we have the issue number, or `// per LWG issue filed on 2021-MM-DD` if we have the date, or `// per LWG issue filed in August 2021` in the absence of a precise date?

The fact that an LWG issue has been submitted and that this constraint is obviously beneficial is good enough for me, though. In general, we should always try to comment when we're significantly deviating from the Standardese as written, but I'm not worried about this particular case and would be fine with or without a comment.